packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -146,6 +146,24 @@ and any changes you make to it will be reflected in the resulting template:
 output.value = cfnBucket.attrArn;
 ```
 
+## Parameters
```

Comment:
Can you actually move this section to be right above the existing "Conditions" section?